js/main.js

```diff
@@ -0,0 +1,40 @@
+function Register() {
+  this.domElement = null;
+  this.total = 0.00;
+
+  this.initialize = function(cont) {
+    this.domElement = $('<div>');
+    cont.append(this.domElement);
+  }
+
+  this.render = function() {
```

Comment:
Right now every time you create a new Register it will create a new copy of all of these functions as well, this could be avoided by converting these into functions on the prototype
```js
Register.prototype.render = function() {
```